app/Http/Controllers/Api/v1/EntityApiController.php

```diff
@@ -2,19 +2,22 @@
 
 namespace App\Http\Controllers\Api\v1;
 
+use App\Http\Requests\API\PatchEntity;
 use App\Http\Requests\API\StoreEntities;
 use App\Http\Resources\EntityResource as Resource;
 use App\Models\Campaign;
 use App\Models\Entity;
 use App\Models\EntityType;
 use App\Services\Api\BulkEntityCreatorService;
+use App\Services\Entity\TransformService;
```

Comment:
I can't figure out where/why this is included

Reply:
Forgot to remove it